app/helpers/application_helper.rb

```diff
@@ -55,4 +55,8 @@ def twitter_url
   def unconfirmed_email?
     logged_in? && current_user.unconfirmed?
   end
+
+  def format_markdown(str)
+    CommonMarker.render_html(str, :GITHUB_PRE_LANG, [:tagfilter, :autolink, :table, :strikethrough]).gsub(/(\\n|\\r)/, '<br>').html_safe
```

Comment:
The only thing I'm concerned about is the html_safe here as it's going to be user generated content and could open up xss security issues

Reply:
I've looked into it more, common marker strips out all the bad stuff by default, ignore my last comment